modules/webauthorprofile/lib/webauthorprofile_webinterface.py

```diff
@@ -398,13 +398,19 @@ def index(self, req, form):
         profile_page = WebProfilePage("profile", webapi.get_longest_name_from_pid(self.person_id))
         profile_page.add_profile_menu(menu)
 
+        if 'form_email' in pinfo:
+            gFormEmail = pinfo['form_email']
```

Comment:
What do you mean  by `correctly available`?

If sb was a `guest` and sent a ticket, I memorize it. After this if he authenticates, I don't care about the memorized ticket, so the one provided in the profile will be added to the ticket.